api/src/api/app/controller/scenario_controller.py

```diff
@@ -118,7 +118,7 @@ async def import_scenario_data(
         file: UploadFile,
     ) -> ID:
         """Supply simulation data for a scenario."""
-        if not file or not file.filename.endswith('.zip'):
+        if not file or not file.filename or not file.filename.endswith('.zip'):
```

Comment:
does not belong to this feature. linting stuff.
```suggestion
        if not file or not file.filename.endswith('.zip'):
```